src/components/DatePicker/index.android.js

```diff
@@ -1,94 +1,86 @@
+        if (!isFunction(innerRef)) {
+            return;
+        }
+        if (element && element.focus && typeof element.focus === 'function') {
+            let inputRef = {...element};
+            inputRef = {...inputRef, focus: showPicker};
+            innerRef(inputRef);
+            return;
+        }
 
-    render() {
-        const dateAsText = this.props.value || this.props.defaultValue ? moment(this.props.value || this.props.defaultValue).format(CONST.DATE.MOMENT_FORMAT_STRING) : '';
+        innerRef(element);
+    };
 
-        return (
-            <>
-                <TextInput
-                    label={this.props.label}
-                    accessibilityLabel={this.props.label}
-                    accessibilityRole={CONST.ACCESSIBILITY_ROLE.TEXT}
-                    value={dateAsText}
-                    forceActiveLabel
-                    placeholder={this.props.placeholder}
-                    errorText={this.props.errorText}
-                    containerStyles={this.props.containerStyles}
-                    textInputContainerStyles={this.state.isPickerVisible ? [styles.borderColorFocus] : []}
-                    onPress={this.showPicker}
-                    editable={false}
-                    disabled={this.props.disabled}
-                    onBlur={this.props.onBlur}
-                    ref={(el) => {
-                        if (!_.isFunction(this.props.innerRef)) {
-                            return;
-                        }
-                        if (el && el.focus && typeof el.focus === 'function') {
-                            let inputRef = {...el};
-                            inputRef = {...inputRef, focus: this.showPicker};
-                            this.props.innerRef(inputRef);
-                            return;
-                        }
+    const dateAsText = value || defaultValue ? moment(value || defaultValue).format(CONST.DATE.MOMENT_FORMAT_STRING) : '';
 
-                        this.props.innerRef(el);
-                    }}
+    return (
+        <>
+            <TextInput
+                label={label}
+                accessibilityLabel={label}
+                accessibilityRole={CONST.ACCESSIBILITY_ROLE.TEXT}
+                value={dateAsText}
+                forceActiveLabel
+                placeholder={placeholder}
+                errorText={errorText}
+                containerStyles={containerStyles}
+                textInputContainerStyles={isPickerVisible ? [styles.borderColorFocus] : []}
+                onPress={() => showPicker()}
+                editable={false}
+                disabled={disabled}
+                onBlur={onBlur}
+                ref={(element) => setRef(element)}
+            />
+            {isPickerVisible && (
+                <RNDatePicker
+                    value={value || defaultValue ? moment(value || defaultValue).toDate() : new Date()}
+                    mode="date"
+                    onChange={() => setDate()}
```

Comment:
Did you test android app after your last commit? `setDate` requires 2 params.

Please add screenshot, at least for android.

Reply:
Oh, so sorry about that :/
Just fixed it and attached a evidence for Android